src/os/bluestore/BlueStore.cc

```diff
@@ -5050,8 +5050,8 @@ void BlueStore::_init_logger()
   b.add_time_avg(l_bluestore_state_kv_queued_lat, "state_kv_queued_lat",
 		"Average kv_queued state latency",
 		"skql", PerfCountersBuilder::PRIO_USEFUL);
-  b.add_time_avg(l_bluestore_state_kv_committing_lat, "state_kv_commiting_lat",
-		 "Average kv_commiting state latency",
+  b.add_time_avg(l_bluestore_state_kv_committing_lat, "state_kv_committing_lat",
+		 "Average kv_committing state latency",
```

Comment:
Same as above, I'm inclined to accept this with an NB in the release notes.  @zdover23 @epuertat